Docs/releases/release-3.5.0.md

```diff
@@ -26,6 +26,7 @@ You will need to update your Creator Companion to version 2.2.0 in order to mana
 * Switches from .NET Framework 4.x to .NET Standard 2.1.
 * Restores UdonSharp Samples, now  listed under World Samples.
 * Restores UdonSharp and ClientSim legacyFiles.
+* Now you can add `VRC Imposter Settings` Component!
```

Comment:
How about rephrasing this, fixing the typo, and linking to the docs?
```suggestion
* Adds the [VRC Impostor Settings](/avatars/avatar-impostors#vrcimpostorsettings) component to the avatars SDK.
```

Reply:
I think linking to the docs is good but "Adds" is incorrect I think.

Reply:
That sentence looks grammatically correct to me! It's in the present tense, like the other lines from the release notes.

Reply:
I means "Adds" is not correct fact, not incorrect in grammatically. Imposter Settings was exists in VRCSDK 3.3.0 (or older).

Reply:
I see. This this line be added to the 3.3.0 release notes then? Or are you trying to communicate something else?

Sorry for the misunderstanding.

Reply:
I added that to 3.3.0 release note